src/main/java/org/aksw/simba/lemming/CompatibleGraph.java

```diff
@@ -0,0 +1,22 @@
+package org.aksw.simba.lemming;
+
+
+import org.dice_research.ldcbench.graph.Graph;
+import org.dice_research.ldcbench.graph.GrphBasedGraph;
+
+import grph.Grph;
+
+public class CompatibleGraph extends GrphBasedGraph {
```

Comment:
It was a work-around :sweat: I needed the Grph object in [GrphBasedGraph](https://github.com/dice-group/orca/blob/c01c6ac7dbd76d551b073c70b3d1c807bd58a677/ldcbench.api/src/main/java/org/dice_research/ldcbench/graph/GrphBasedGraph.java#L26) to convert from the barabasi graph to a coloured graph. I needed a getter and couldn't find one, but the field was set to protected so that ended up being the reason for this class. 

I've renamed it (hopefully) into something more meaningful.